.github/workflows/intergration-test.yml

```diff
@@ -0,0 +1,35 @@
+name: "Integration Test"
+
+on: [push, pull_request]
```

Comment:
on `pull_request` cannot access secrets.

We need to reach a consensus on this problem https://github.com/beyondstorage/go-service-example/issues/6#issuecomment-875844542

Reply:
I have changed the trigger event to `pull_request_target`,  add any labels to have a test.